assets/stylesheets/components/_forms.scss

```diff
@@ -0,0 +1,254 @@
+    color: $input-color;
+    padding: 0 20px;
+    border-radius: $input-border-radius;
+    height: $input-height-base;
+
+    &:focus {
+      box-shadow: none;
+    }
+  }
+}
+
+.radio,
+.checkbox {
+  label {
+    padding-left: 25px;
+  }
+
+  input[type="radio"],
+  input[type="checkbox"] {
+    margin-left: -25px;
+  }
+}
+
+input[type="radio"]:disabled,
+input[type="checkbox"]:disabled {
+  -webkit-appearance: none;
+  content: '';
+  width: 15px;
+  height: 15px;
+  background-color: $cr-gray;
+  outline: 0;
+}
+
+input[type="radio"]:disabled {
+  border-radius: 50%;
+}
+
+input[type="checkbox"]:disabled {
+  border-radius: 4px;
+}
+
+.helper-text {
+  color: $cr-gray;
+  font-size: .8125rem; // 13px
+  padding-top: 8px;
+}
+
+.input-group {
+  input {
+    padding: 0 10px;
+  }
+
+  .input-group-addon {
+    background: $input-group-addon-bg;
+    border: $input-border;
+    border-radius: $input-border-radius;
+    color: $input-group-addon-border-color;
+    font-size: 1.125rem;
+  }
+}
+
+// Datepicker
+
+datepicker {
+  $calendar-bg: $cr-white;
+  $calendar-text: $cr-gray-darker;
+  $calendar-border: $cr-gray-light;
+  $calendar-active-bg: $cr-teal;
+  $calendar-active-text: $cr-white;
+  $calendar-active-border: $cr-teal;
+  $calendar-hover-bg: $cr-gray-lighter;
+  $calendar-hover-text: $cr-gray-darker;
+  $calendar-hover-border: $cr-gray;
+
+  display: none;
+
+  .well {
+    margin-bottom: 0;
+    margin-top: -.5em;
+  }
+
+  &.visible {
+    display: block;
+  }
+
+  th {
+    .btn-default {
+      &.btn-secondary {
+        &.btn-sm {
+          border: 0;
+          border-radius: 0;
+          width: 100%;
+
+          &.pull-left {
+            border-top-left-radius: 4px;
+            border-bottom-left-radius: 4px;
+            border-top-right-radius: 0;
+            border-bottom-right-radius: 0;
+          }
+
+          &.pull-right {
+            border-top-left-radius: 0;
+            border-bottom-left-radius: 0;
+            border-top-right-radius: 4px;
+            border-bottom-right-radius: 4px;
+          }
+        }
+      }
+    }
+  }
+
+  .btn {
```

Comment:
Ah – I see below that you've killed `_buttons.scss` and move those styles into here. Can you talk to the benefit you in doing that? Is that how Bootstrap has architected its styles too?